packages/esm-patient-chart-app/src/visit/visits-widget/past-visits-components/visits-table/visits-table.component.tsx

```diff
@@ -295,7 +295,7 @@ const VisitTable: React.FC<VisitTableProps> = ({ showAllEncounters, visits, pati
                                   className={styles.menuItem}
                                   itemText={t('deleteThisEncounter', 'Delete this encounter')}
                                   onClick={() => {
-                                    handleDeleteEncounter(visits[index]?.id, visits[index]?.form?.display);
+                                    handleDeleteEncounter(row.id, row.form?.display)
```

Comment:
I don't think `row` has a `form` or `display` property. Did you try this?

Reply:
@ibacher You are correct. In HTML, there is no element called <row>, and therefore, it does not have any form or display property associated with it.

Reply:
@ibacher Kindly I didn't understand this question` Did you try this?`